### PR TITLE
replace old repo refs

### PR DESCRIPTION
**Pull request by akselleirv**
Relates to https://github.com/flux-iac/tofu-controller/issues/1200

Replacements:
- weaveworks -> flux-iac
- tf-controller -> tofu-controller
- Terraform controller -> Tofu controller
- Terraform Controller -> Tofu Controller

### CODE REVIEW

File: .github/workflows/blocked-reminder.yaml [hunk 1]
```diff
@@ -79,7 +79,7 @@ jobs:
         uses: archive/github-actions-slack@d9dae40827adf93bddf939db6552d1e392259d7d # v2.7.0
         if: ${{ steps.list-issues.outputs.result != '' }}
         with:
-          slack-bot-user-oauth-access-token: ${{ secrets.WEAVEWORKS_SLACK_GENERICBOT_TOKEN }}
```
Comment (akselleirv) [inline .github/workflows/blocked-reminder.yaml:82]: @chanwit should this be reverted or fixed in the repo secrets?

Comment (chanwit) [reply .github/workflows/blocked-reminder.yaml:82]: Please remove the whole `.github/workflows/blocked-reminder.yaml` file.
It was for an internal team to get notified about blocking issues.

**Comment by akselleirv** [comment]
@chanwit should the `tf-runner` also be renamed to `tofu-runner`?

**Comment by chanwit** [comment]
Let's leave it like that for a while.

We still support TF and will add tofu-runner images later for example

**Comment by akselleirv** [comment]
I'm gonna split this PR into smaller ones to avoid merge conflicts with others.

**Pull request closed without merging**